test/DynamoCoreWpfTests/SliderViewModelTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Windows.Controls;
+using CoreNodeModels;
+using CoreNodeModels.Input;
+using CoreNodeModelsWpf;
+using CoreNodeModelsWpf.Controls;
+using Dynamo.Nodes;
+using Dynamo.Tests;
+using Dynamo.Utilities;
+using DynamoCoreWpfTests.Utility;
+using NUnit.Framework;
+
+namespace DynamoCoreWpfTests
+{
+    [TestFixture]
+    class SliderViewModelTests : DynamoTestUIBase
+    {
+        private AssemblyHelper assemblyHelper;
+
+        public override void Open(string path)
+        {
+            base.Open(path);
+
+            DispatcherUtil.DoEvents();
```

Comment:
Do we do the override because we need to append the `DoEvents()` call?

Reply:
@QilongTang 
Yes, you are right, the override is because we need to call the DoEvents() method so all the events in the graph are executed. I have seen the same override in many tests.
Thanks